bazel/install-deps.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# Copyright 2020 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+set -e
+
+echo "installing redpanda toolchain"
+
+if [[ $EUID -ne 0 ]]; then
+  echo "This script should be run as root."
+  exit 1
+fi
+if [ -f "/etc/os-release" ]; then
+  . /etc/os-release
+elif [ -f "/etc/arch-release" ]; then
+  export ID=arch
+else
+  echo "/etc/os-release missing."
+  exit 1
+fi
+
+deb_deps=(
+  clang
+  lld
+  ragel
+  automake
+  make
+  autoconf
+  libtool
+  autopoint
+  xfslibs-dev
+  valgrind
+  git
+  cargo
+  rustc
+  bison
```

Comment:
to build this on my freshly installed ubuntu noble I also needed `xutils-dev` for libpciaccess

Reply:
How about this https://github.com/dotnwat/redpanda/pull/20 ?